### PR TITLE
[aws/slack-archive] added new module

### PR DESCRIPTION
**Pull request by maximmi**
## what
* module to create infrastructure for slack archive website

## why
* we need static Slack archive website

### CODE REVIEW

File: aws/slack-archive/main.tf [hunk 1]
```diff
@@ -0,0 +1,122 @@
+terraform {
+  required_version = ">= 0.11.2"
+
+  backend "s3" {}
+}
+
+variable "aws_assume_role_arn" {
+  type = "string"
```
Comment (aknysh) [inline aws/slack-archive/main.tf]: add description

File: aws/slack-archive/main.tf [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+variable "domain_name" {
+  type = "string"
```
Comment (aknysh) [inline aws/slack-archive/main.tf]: add description

File: aws/slack-archive/main.tf [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+variable "namespace" {
+  type        = "string"
+  description = "Namespace (e.g. `cp` or `cloudposse`)"
```
Comment (aknysh) [inline aws/slack-archive/main.tf]:
```suggestion
  description = "Namespace (e.g. `eg` or `cp`)"
```

File: aws/slack-archive/outputs.tf [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+output "slack_archive_user_unique_id" {
+  value       = "${module.slack_archive_user.user_unique_id}"
+  description = "The user unique ID assigned by AWS"
+}
+
+output "slack_archive_user_access_key_id" {
+  value       = "${module.slack_archive_user.access_key_id}"
+  description = "The access key ID"
```
Comment (aknysh) [inline aws/slack-archive/outputs.tf]: add
```
sensitive = true
```

https://www.terraform.io/docs/configuration/outputs.html

File: aws/slack-archive/outputs.tf [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+output "slack_archive_user_secret_access_key" {
+  value       = "${module.slack_archive_user.secret_access_key}"
+  description = "The secret access key. This will be written to the state file in plain-text"
```
Comment (aknysh) [inline aws/slack-archive/outputs.tf]: add
```
sensitive = true
```

https://www.terraform.io/docs/configuration/outputs.html

File: aws/slack-archive/terraform.tfvars.example [hunk 6]
```diff
@@ -0,0 +1,4 @@
+namespace="cp"
```
Comment (aknysh) [inline aws/slack-archive/terraform.tfvars.example]:
```suggestion
namespace="eg"
```

File: aws/slack-archive/main.tf (hunk 1 above)
Comment (maximmi) [reply aws/slack-archive/main.tf]: done

File: aws/slack-archive/main.tf (hunk 2 above)
Comment (maximmi) [reply aws/slack-archive/main.tf]: done

File: aws/slack-archive/main.tf (hunk 3 above)
Comment (maximmi) [reply aws/slack-archive/main.tf]: done

File: aws/slack-archive/outputs.tf (hunk 5 above)
Comment (maximmi) [reply aws/slack-archive/outputs.tf]: done

File: aws/slack-archive/terraform.tfvars.example (hunk 6 above)
Comment (maximmi) [reply aws/slack-archive/terraform.tfvars.example]: done

File: aws/slack-archive/outputs.tf (hunk 4 above)
Comment (maximmi) [reply aws/slack-archive/outputs.tf]: done

**Pull request merged**